packages/backend/src/plugins/search.ts

```diff
@@ -73,7 +73,14 @@ export default async function createPlugin({
 
   indexBuilder.addCollator({
     defaultRefreshIntervalSeconds: 600,
-    collator: new DefaultTechDocsCollator({ discovery, logger }),
+    collator: new DefaultTechDocsCollator({
```

Comment:
Good opportunity to create a static `DefaultTechDocsCollator::fromConfig()` equivalent?  I think catalog recently updated to do the same.

Reply:
:+1: I'll add a fromConfig static method and a deprecation flag to the constructor